client/blocks/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import {
+	registerPaymentMethod,
+	registerExpressPaymentMethod,
+} from '@woocommerce/blocks-registry';
+
+/**
+ * Internal dependencies
+ */
+import stripeCcPaymentMethod from './credit-card';
+import paymentRequestPaymentMethod from './payment-request';
+import { getStripeServerData } from './stripe-utils';
+
+// Register Stripe Credit Card.
+registerPaymentMethod( stripeCcPaymentMethod );
+
+// Register Stripe Payment Request (Apple/Chrome Pay) if enabled.
+// Make sure `allowPaymentRequest` defaults to false if it's missing from the server
+// provided configuration.
+if ( getStripeServerData().allowPaymentRequest ?? false ) {
+	registerExpressPaymentMethod( paymentRequestPaymentMethod );
+}
```

Comment:
This prevents the Apple Pay button preview from showing up in the editor context. No need to fix now, but just as a note.

We need to always register it with`registerExpressPaymentMethod` to allow for the preview.

Reply:
This is an awesome catch! 👏